aks/postgres/variables.tf

```diff
@@ -121,3 +121,8 @@ variable "azure_enable_backup_storage" {
   type    = bool
   default = true
 }
+
+variable "azure_enable_backing_services" {
+  type    = bool
+  default = false
+}
```

Comment:
We don't need this as we've got `use_azure` and `azure_enable_monitoring` already.

Reply:
as per above comment